application/util/string.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import re
+
+phone_number_re = re.compile('(\\+\\d+)? \\((\\d\\d\\d)\\) (\\d\\d\\d)-(\\d\\d\\d\\d)')
+
+
+def reverse_string(x):
+    return reverse_string(x[1:] + x[0])
```

Comment:
...is this a recursive function call _in_ the function? didn't know it could do that!